project/scenes/game/characters/player/States/ClimbingState.gd

```diff
@@ -1,8 +1,7 @@
 class_name ClimbingState extends State
 
-@export var VERTICAL_SPEED :float = 400
-
-@onready var csm = $".."
+var jump_on_wall_x_velocity = 400
+var jump_on_wall_y_velocity = 200
```

Comment:
bi mogli imet en central script za player, kjer imamo vse variables. Splitat v različne file, se samo iščeš

```
player_config.gd

ClimbingState.gd -> autoload player_config.gd
```

Reply:
te niso propriamente variabili, rabijo samo za "jump on wall animation", ki je se kinda work in progress. Sem jih pustil tle tku da ne clutteram variables file. Mislim da jih bomo zbrisali, in dali semplicemente fixed value na jump on wall kadar bo koncan. 

Za zdej jih pustim tam